README.md

```diff
@@ -5,7 +5,10 @@
 
 Welcome to Bridge-To-Kubernetes! Bridge to Kubernetes extends the Kubernetes perimeter to your development computer allowing you to write, test, and debug microservice code while connected to your Kubernetes cluster with the rest of your application or services. You can simply run your code natively on your development workstation while connected to the Kubernetes cluster, allowing you to test your code changes in the context of the larger application.
 
+## Introduction Video
+https://learn.microsoft.com/en-us/shows/open-at-microsoft/get-started-with-bridge-to-kubernetes
 
+![image002](https://github.com/Azure/Bridge-To-Kubernetes/assets/105889062/7d71f41d-dafe-4039-afb1-31600f4a793f)

```

Comment:
could do this for guidance: https://stackoverflow.com/a/14945782/1210059

code : `[![Watch the video](https://github.com/Azure/Bridge-To-Kubernetes/assets/105889062/7d71f41d-dafe-4039-afb1-31600f4a793f)](https://www.youtube.com/watch?v=IkLqFoFsV40&list=PLlrxD0HtieHg8On6t1l5_kj--7PMmyfGi&index=22)` 

[![Watch the video](https://github.com/Azure/Bridge-To-Kubernetes/assets/105889062/7d71f41d-dafe-4039-afb1-31600f4a793f)](https://www.youtube.com/watch?v=IkLqFoFsV40&list=PLlrxD0HtieHg8On6t1l5_kj--7PMmyfGi&index=22)